docs/package.json

```
@@ -30,6 +30,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
     "@material-ui/core": "5.0.0-alpha.23",
+    "@material-ui/data-grid": "^4.0.0-alpha.15",
```

I had this in mind, but we can also wait for the next releast

```suggestion
    "@material-ui/data-grid": "https://pkg.csb.dev/mui-org/material-ui-x/commit/268ed0d0/@material-ui/data-grid",
```

```suggestion
    "@material-ui/data-grid": "^4.0.0-alpha.18",
```

---

Do you plan on replacing this before merge with a version published to npm?

---

Yes, I'll keep it as a draft until then.